e2e/cypress/integration/system_console/MM_27188_System_Console_spec.js

```diff
@@ -0,0 +1,416 @@
+            });
+
+            cy.findByTestId('totalSessionsTitle').then((el) => {
+                const titleSpan = el[0].childNodes[0];
+                expect(titleSpan.scrollWidth > titleSpan.clientWidth).equal(false);
+            });
+
+            cy.findByTestId('totalCommandsTitle').then((el) => {
+                const titleSpan = el[0].childNodes[0];
+                expect(titleSpan.scrollWidth > titleSpan.clientWidth).equal(true);
+            });
+
+            cy.findByTestId('incomingWebhooksTitle').then((el) => {
+                const titleSpan = el[0].childNodes[0];
+                expect(titleSpan.scrollWidth > titleSpan.clientWidth).equal(false);
+            });
+
+            cy.findByTestId('outgoingWebhooksTitle').then((el) => {
+                const titleSpan = el[0].childNodes[0];
+                expect(titleSpan.scrollWidth > titleSpan.clientWidth).equal(false);
+            });
+
+            cy.findByTestId('dailyActiveUsersTitle').then((el) => {
+                const titleSpan = el[0].childNodes[0];
+                expect(titleSpan.scrollWidth > titleSpan.clientWidth).equal(false);
+            });
+
+            cy.findByTestId('monthlyActiveUsersTitle').then((el) => {
+                const titleSpan = el[0].childNodes[0];
+                expect(titleSpan.scrollWidth > titleSpan.clientWidth).equal(false);
+            });
+
+            cy.findByTestId('websocketConnsTitle').then((el) => {
+                const titleSpan = el[0].childNodes[0];
+                expect(titleSpan.scrollWidth > titleSpan.clientWidth).equal(false);
+            });
+
+            cy.findByTestId('masterDbConnsTitle').then((el) => {
+                const titleSpan = el[0].childNodes[0];
+                expect(titleSpan.scrollWidth > titleSpan.clientWidth).equal(true);
+            });
+
+            cy.findByTestId('replicaDbConnsTitle').then((el) => {
+                const titleSpan = el[0].childNodes[0];
+                expect(titleSpan.scrollWidth > titleSpan.clientWidth).equal(true);
+            });
+        });
+    });
+
+    // This test assumes EE license already uploaded!
+    it('MM-T1201 - Remove and re-add license - Permissions freeze in place when license is removed (and then re-added)', () => {
+        // # Go to admin console and set permissions as listed in the test
+        goToAdminConsole();
+        cy.visit('admin_console/user_management/permissions/system_scheme');
+        cy.findByTestId('resetPermissionsToDefault').click();
+        cy.get('#confirmModalButton').click();
+        cy.findByTestId('all_users-public_channel-create_public_channel-checkbox').click();
+        cy.findByTestId('all_users-private_channel-manage_private_channel_properties-checkbox').click();
+        cy.findByTestId('team_admin-private_channel-manage_private_channel_properties-checkbox').click();
+        cy.findByTestId('saveSetting').click();
+
```

Comment:
The cypress test currently misses a very important step, i.e. to remove the EE license and then re-upload it after the testcase is completed.

Reply:
Ok done! I don't reupload it but I do upload it initially if not already uploaded and since this is the last test case in the spec, it doesn't matter as long as other spec files do upload it (`requireLicense` function called then we good)